src/sexpr/sexpr.ts

```diff
@@ -8,6 +8,7 @@ export enum STypes {
   Boolean,
   Nil,
   List,
+  Boxed,
```

Comment:
This is referencing: https://docs.racket-lang.org/reference/boxes.html right? Why do we include this?

Reply:
no actually it wasn't, i guess i should rename it :thinking: 

it's supposed to be a leaf node that can extend the s expression type easily to support other kinds of leaves. by default, s expressions only support 3 leaf nodes, atom, boolean, and number.

Reply:
What other types of leaf nodes would we want to support with this?